### DISCUSSION TITLE
DiscordChatExporter.Domain.Exceptions.DiscordChatExporterException: Authentication token is invalid. ERROR

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [Tyrrrz/DiscordChatExporter#924] [Q&A]
Hello, I'm trying to export my old server's chat, but I'm getting DiscordChatExporter.Domain.Exceptions.DiscordChatExporterException: Authentication token is invalid. Error but I took the token true and tried it with lots of servers, but I'm still getting this error. How can I solve it?

**Comment by Tyrrrz** [comment]
So the token doesn't work for only one server?
